### PR TITLE
DOCS-3875:Add Magento 2 Hyva Checkout & Megawire modules

### PR DESCRIPTION
**Pull request by Rendani-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/integrations/magento-2.md [hunk 1]
```diff
@@ -123,11 +123,37 @@ The plugin is compatible with most Magento checkouts.
...
+    <td> <a href="https://www.mageplaza.com/magento-2-one-step-checkout-extension" target="_blank">MagePlaza One Step Checkout</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i></td>
+    <td> MagePlaza One Step module not available.</td>
+  </tr>
+  <tr>
+    <td> <a href="https://hyva.io/hyva-checkout.html" target="_blank"> Hyvä Megawire Checkout</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i></td>
+    <td><a href="https://github.com/MultiSafepay/magewire-checkout" target="_blank">Magewire Module</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i></td>
+  </tr>
+  <tr>
+    <td><a href="https://www.onestepcheckout.com/" target="_blank">OneStepCheckout.com</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i></td>
+    <td>One Step Module not available</td>
```
Comment (Daniel-MultiSafepay) [inline content/integrations/magento-2.md:151]: This is supposed to say it is supported by default, or out of the box.

File: content/integrations/magento-2.md [hunk 2]
```diff
@@ -123,11 +123,37 @@ The plugin is compatible with most Magento checkouts.
...
-- <a href="https://www.onestepcheckout.com/" target="_blank">OneStepCheckout.com</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i>: MultiSafepay is an official partner, and the plugin is compatible with the latest version of the OneStepCheckout extension. Always test first to check compatibility with your configuration.
+<table>
+  <tr>
+    <th>Checkout</th>
+    <th>Module</th>
+   
+  </tr>
+  <tr>
+    <td> <a href="https://amasty.com/one-step-checkout-for-magento-2.html" target="_blank">Amasty One Step Checkout</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i></td>
+    <td> Amasty One Step module not available.</td>
```
Comment (Daniel-MultiSafepay) [inline content/integrations/magento-2.md:134]: This is supposed to say it is supported by default, or out of the box.

**Review by Daniel-MultiSafepay — CHANGES_REQUESTED** [review]

File: content/integrations/magento-2.md [hunk 3]
```diff
@@ -123,11 +123,37 @@ The plugin is compatible with most Magento checkouts.
...
+    <th>Module</th>
+   
+  </tr>
+  <tr>
+    <td> <a href="https://amasty.com/one-step-checkout-for-magento-2.html" target="_blank">Amasty One Step Checkout</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i></td>
+    <td> Amasty One Step module is supported by default.</td>
+    
+  </tr>
+  <tr>
+    <td> <a href="https://hyva.io/hyva-checkout.html" target="_blank"> Hyvä Checkout</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i></td>
```
Comment (Daniel-MultiSafepay) [inline content/integrations/magento-2.md:138]: This is wrong. You are matching Hyva React checkout to Magewire Checkout using this link. 

Please, could you check my comments here: 

https://jira.multisafepay.com/browse/DOCS-3875?focusedCommentId=171820&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-171820

**Review by Daniel-MultiSafepay — CHANGES_REQUESTED** [review]
Check my comments.

File: content/integrations/magento-2.md [hunk 4]
```diff
@@ -122,12 +122,43 @@ The plugin is compatible with most Magento checkouts.
...
+  <tr>
+    <td><a href=" https://github.com/hyva-themes/magento2-react-checkout" target="_blank"> Hyvä React checkout </a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i>  </td>
+    <td><a href="https://github.com/MultiSafepay/magento2-hyva-checkout-multisafepay" target="_blank"> Hyvä checkout module</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i></td>
+  </tr>
+  <tr>
+    <td><a href="https://www.mageplaza.com/magento-2-one-step-checkout-extension" target="_blank">MagePlaza One Step checkout</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i> </td>
+    <td>By default,MagePlaza One Step checkout is supported. </td>
+  </tr>
+  <tr>
+    <td><a href=" https://docs.hyva.io/checkout/hyva-checkout/index.html " target="_blank"> Hyvä Megawire checkout</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i> </td>
```
Comment (Daniel-MultiSafepay) [inline content/integrations/magento-2.md:151]: Fix empty whitespaces in the href tag.

File: content/integrations/magento-2.md [hunk 5]
```diff
@@ -122,12 +122,43 @@ The plugin is compatible with most Magento checkouts.
...
+    <td><a href="https://amasty.com/one-step-checkout-for-magento-2.html" target="_blank">Amasty One Step Checkout</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i></td>
+    <td>By default, the Amasty One Step checkout is supported.</td>
+  </tr>
+  <tr>
+    <td><a href=" https://github.com/hyva-themes/magento2-react-checkout" target="_blank"> Hyvä React checkout </a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i>  </td>
+    <td><a href="https://github.com/MultiSafepay/magento2-hyva-checkout-multisafepay" target="_blank"> Hyvä checkout module</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i></td>
+  </tr>
+  <tr>
+    <td><a href="https://www.mageplaza.com/magento-2-one-step-checkout-extension" target="_blank">MagePlaza One Step checkout</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i> </td>
+    <td>By default,MagePlaza One Step checkout is supported. </td>
```
Comment (Daniel-MultiSafepay) [inline content/integrations/magento-2.md:148]: The space after the comma is missed.

**Review by Daniel-MultiSafepay — CHANGES_REQUESTED** [review]
Check comments.

**Review by Daniel-MultiSafepay — DISMISSED** [review]

**Review by Daniel-MultiSafepay — CHANGES_REQUESTED** [review]
Remove block commented or explain why you need to keep it

File: content/integrations/magento-2.md [hunk 6]
```diff
@@ -122,12 +122,43 @@ The plugin is compatible with most Magento checkouts.
 <details id="supported-checkouts">
 <summary>Supported checkouts</summary>
 <br>
-
+<!--
```
[the remark this answers is not in the document]
Comment (Rendani-MultiSafepay) [reply content/integrations/magento-2.md:125]: Hi, I keep the uncommented block if ReadMe does not interpret the HTML table tags. That way, I can quickly use the uncommented block to update the checkouts on the browser. However, should the table be interpreted right? I will remove the block.

[the remark this answers is not in the document]
Comment (Daniel-MultiSafepay) [reply content/integrations/magento-2.md:125]: I have no idea how the platform used for DOCS renders the code, but as far as I know, some HTML tags are going to be stripped, but not sure which ones. 

It seems we are using markdown syntax for tables. I can't find any table being done using HTML tags. 

Please, check this as example:
- https://github.com/MultiSafepay/docs/blob/master/content/integrations/opencart.md?plain=1#L70-L82

Reference:
- https://www.markdownguide.org/extended-syntax/#tables

**Review by Daniel-MultiSafepay — APPROVED** [review]

File: content/integrations/magento-2.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (Daniel-MultiSafepay) [reply content/integrations/magento-2.md:125]: Understood. I just approved this PR. 

Is up to you if you want to remove the block commented now or later, depending on the results after the merge

**Pull request merged**